modules/integration/src/test/scala/scala/cli/integration/HelpTests.scala

```diff
@@ -46,6 +46,31 @@ class HelpTests extends ScalaCliSuite {
       expect(fullHelpOutput.contains("Legacy Scala runner options"))
     }
   }
+
+  def getOptionAliasesForExtraJars(help: String): Seq[String] = {
+    val description = "Add extra JARs and compiled classes to the class path"
+    expect(help.contains(description))
+    val line = help.linesIterator
+      .find(_.contains(description))
+      .get
```

Comment:
Can you add some logging to better understand what's happening? that the description should be updated to reflect the value that is set in the HelpMessage.

Reply:
I simplified the test so this function is now removed